docs/content/intro/quickstart.md

```diff
@@ -0,0 +1,314 @@
+---
+title: A Brigade Quickstart
+description: A Brigade Quickstart.
+section: intro
+---
+
+## Install Brigade
+
+The easiest way to install Brigade into your Kubernetes cluster is to install it using [Helm](https://helm.sh/), the Kubernetes Package Manager.
+
+```bash
+# add Brigade chart repo
+helm repo add brigade https://brigadecore.github.io/charts
+# install Brigade
+helm install -n brigade brigade/brigade
+
+# if you want to activate Generic Gateway, you should use this command
+# helm install -n brigade brigade/brigade --set genericGateway.enabled=true
+```
+
+You will now have Brigade installed. [Kashti](https://github.com/brigadecore/kashti), the dashboard for your Brigade pipelines, is also installed in the cluster.
+
+## Install brig
+
+Brig is the Brigade command line client. You can use `brig` to create/update/delete new brigade Projects, run Builds, etc. To get `brig`, navigate to the [Releases page](https://github.com/brigadecore/brigade/releases/) and then download the appropriate client for your platform. For example, if you're using Linux or WSL, you can get the 0.20.0 version in this way:
+
+```bash
+wget -O brig https://github.com/brigadecore/brigade/releases/download/v0.20.0/brig-linux-amd64
+chmod +x brig
+mv brig ~/bin
+```
+
+Alternatively, you can use [asdf-brig](https://github.com/Ibotta/asdf-brig) to install & manage multiple versions of `brig`.
+
+We have two quickstarts for you to check.
+- The first one creates a Project that will pull source from a Version Control System (VCS). This is usually used as a CI/CD pipeline.
+- The second one creates a Project that has no dependency on a Version Control System and its Builds will be triggered via POST requests on Brigade's [Generic Gateway](https://docs.brigade.sh/topics/genericgateway/). Think of this approach as some JavaScript code that will do stuff with containers and be triggered by a POST message (either a plain JSON one or a [CloudEvent](https://cloudevents.io/)).
+
+By the way, this does not mean that you cannot combine these scenarios (i.e. have your Builds triggered by POST requests and your source code be pulled and acted upon) or think of alternative ways to use Brigade!
+
+## Using Brigade with a Version Control System
+
+### Creating A New Project - using a Version Control System
+
+To create a new project, use `brig project create` and answer the prompts. Feel free to modify or leave all options at their defaults (just press Enter on every interactive prompt).
+
+```bash
+brig project create
+```
+
+Output would be similar to this:
+```
+? Project name deis/empty-testbed
```

Comment:
Here and elsewhere in this doc, `deis` references can/should now be updated to `brigadecore` (May inherit these changes if rebased off of master?)

Reply:
You're right, but the thing is that the default suggestion by `brig project create` is `deis/empty-testbed`. So user just presses the Enter key and `deis-empty-testbed` is added. So I'm a bit hesitant to change it here before we change it on `brig project create`. WDYT?

Reply:
Ah, I can see the hesitation.  However, the master build of `brig` now does supply `brigadecore` defaults for a project configuration.  This will persist into the v1.0.0 release of brig, so it should be ready for these docs.

Reply:
oh, missed [that](https://github.com/brigadecore/brigade/blob/master/brig/cmd/brig/commands/project_create.go#L59). Will fix!

Reply:
I think it's fixed now, let me know!

Reply:
also changed 0.20.0 to 1.0.0 for `brig` installation

Reply:
Ah, good call 👍